libzhlgen/test/functions/ELF_amd64/1.6.13/CachedRect.zhl

```diff
@@ -0,0 +1,4 @@
+"41544189d45589f5533977104889fb":
+cleanup __amd64 void CachedRect::SetPosition(CachedRect *this, int x, int y);
+"41544189d45589f5533977184889fb":

```

Comment:
These are directly after one another in the binary file, so the second one should have `.` for optimization purposes.

`".41544189d45589f5533977184889fb":`

(I mostly care about this for 64-bit since it's the primary version used for MV, x86 is fine as long as it's a unique match)

Reply:
Done!